internal/validators/validate_test.go

```diff
@@ -0,0 +1,162 @@
+package validators
+
+import (
+	"testing"
+
+	"github.com/go-playground/validator/v10"
+	"github.com/stellar/go/keypair"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestParseValidationError(t *testing.T) {
+	type testStructNested struct {
+		NestedRequiredField string `validate:"required"`
+		NestedEnumField     string `validate:"oneof=foo bar"`
+	}
+
+	type testStruct struct {
+		RequiredField      string             `validate:"required"`
+		RequiredArrayField []string           `validate:"required,gt=0,dive,not_empty"`
```

Comment:
How does the required behave in a non-nullable array field like this one? I see that when it's empty it returns the `gt` message.

Reply:
Resolved async